test/Node/Buffer/Blob.purs

```diff
@@ -0,0 +1,86 @@
+module Test.Node.Buffer.Blob
+  ( test
+  ) where
+
+import Prelude
+
+import Control.Promise as Promise
+import Data.Array (fold)
+import Data.Maybe (Maybe(..))
+import Data.MediaType.Common as MediaTypes
+import Effect (Effect)
+import Effect.Aff (launchAff_)
+import Effect.Class (liftEffect)
+import Effect.Class.Console (log)
+import Node.Buffer.Blob as Blob
+import Test.Assert (assertEqual)
+
+test :: Effect Unit
+test = launchAff_ do
+  log "Testing fromString with no options"
+  testFromStringNoOptions
+  log "Testing fromString with options"
+  testFromStringWithOptions
+  log "Testing fromStrings with no options"
+  testFromStringsNoOptions
+  log "Testing fromStrings with options"
+  testFromStringsWithOptions
+  log "Testing fromArrayBuffer / toArrayBuffer"
+  testToFromArrayBuffer
+  log "Testing size"
+  testSize
+  log "Testing slice"
+  testSlice
+  where
+  testFromStringNoOptions = do
+    let
+      expected = "hello world"
+      blob = Blob.fromString expected Nothing
+    actual <- Promise.toAffE $ Blob.text blob
+    liftEffect $ assertEqual { actual, expected }
+
+  testFromStringWithOptions = do
+    let
+      expected = "{\"hello\":\"world\"}"
+      blob = Blob.fromString expected (Just { "type": MediaTypes.applicationJSON, endings: Blob.Transparent })
+    actual <- Promise.toAffE $ Blob.text blob
+    liftEffect $ assertEqual { actual, expected }
```

Comment:
Shouldn't there be newlines in the expected string? Otherwise, the `Blob.Transparent` part isn't being tested, right?

Reply:
Added newlines.  Not sure though how this option should/could be tested